src/FormattedText.tsx

```diff
@@ -79,6 +79,17 @@ const renderMarkdown = (
     return <div className="format-markdown" dangerouslySetInnerHTML={{ __html }} />;
 }
 
+const createMarkdownLink = (text: string, url: string, target: "_self" | "_blank" = "_self") => {
+  return `[${text}](${url}){:target=${target}}`
+}
+
+const isUrlExternal = (url: string) => !new URL(url).hostname.endsWith(window.location.hostname)
+const determineLinkTarget = (url: string) => isUrlExternal(url) ? "_blank" : "_self"
+
+const determineLinkUrl = (url: string, title: string) => {
+  return title || markdownIt.normalizeLink(url)
```

Comment:
@dvorakjan Prosím Tě, je tohle dobře? Nejsem si jistej, co ten title dělá. Trochu jsme si psali s Honzou a taky moc nevěděl, ale usoudili jsme, že to asi funguje takhle - ale alternativa je, že by ten `title` mohl bejt nějakej `#anchor`, nebo `?query=param`, nebo tak něco... v levym panelu pro srovnání, jak to fungovalo předtím